src/main/java/org/openstreetmap/atlas/geography/atlas/complete/EmptyAtlas.java

```diff
@@ -93,13 +92,13 @@ public Area area(final long identifier)
     }
 
     @Override
-    public GeoJsonObject asGeoJson()
+    public JsonObject asGeoJson()
     {
         throw new UnsupportedOperationException();
     }
 
     @Override
-    public GeoJsonObject asGeoJson(final Predicate<AtlasEntity> matcher)
+    public JsonObject asGeoJson(final Predicate<AtlasEntity> matcher)
     {
         throw new UnsupportedOperationException();
```

Comment:
Please consider returning an Empty equivalent of the json object.

Reply:
I am open to it, but I am following the pattern of the other methods in the EmptyAtlas.java and am not sure about breaking from that convention.

Reply:
EmptyAtlas serves just as a reference to Complete features, so by default, it does not do anything.